File: Code/Server_Prototype/TatueOrganiser/src/model/GuideRating.java
package model;

public class GuideRating {
	private int gr_id = 0;
	private float freundlichkeit = 0;
	private float kompetenz = 0;
	public GuideRating() {
		super();
	}
	public int getGr_id() {
		return gr_id;
	}
	public void setGr_id(int gr_id) {
		this.gr_id = gr_id;
	}
	public float getFreundlichkeit() {
		return freundlichkeit;
	}
	public void setFreundlichkeit(float freundlichkeit) {
		this.freundlichkeit = freundlichkeit;
	}
	public float getKompetenz() {
		return kompetenz;
	}
	public void setKompetenz(float kompetenz) {
		this.kompetenz = kompetenz;
	}
	
}

File: Code/Client_Prototype/Client_Prototype/Classes/GuideRating.cs
namespace BSD_Client
{
    public class GuideRating
    {
        public int GR_ID { get; set; }
        public float freundlichkeit { get; set; }
        public float kompetenz { get; set; }

        public GuideRating()
        {

        }

        public GuideRating(int _ID, float _Freundlichkeit, float _Kompetenz)
        {
            GR_ID = _ID;
            freundlichkeit = _Freundlichkeit;
            kompetenz = _Kompetenz;
        }
    }
}